### PR TITLE
Genome annotation statistics tool: jcvi_gff_stats

### PR DESCRIPTION
**Pull request by abretaud**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

FOR REVIEWER:
* [ ] .shed.yml file ok
    - [ ] Toolshed user `iuc` has access to associated toolshed repo(s)
* [ ] Indentation is correct (4 spaces)
* [ ] Tool version/build ok
* [ ] `<command/>`
  - [ ] Text parameters, input and output files `'single quoted'`
  - [ ] Use of `<![CDATA[ ... ]]>` tags
  - [ ] Parameters of type `text` or having `optional="true"` attribute are checked with `if str($param)` before being used
* [ ] Data parameters have a `format` attribute containing datatypes recognised by Galaxy
* [ ] Tests
  - [ ] Parameters are reasonably covered
  - [ ] Test files are appropriate
* [ ] Help
  - [ ] Valid restructuredText and uses `<![CDATA[ ... ]]>` tags
* [ ] Complies with other best practice in [Best Practices Doc](http://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html)

Hi all,
A new little tool to generate some statistics on (gff) genome annotation (gene, exon and intron number and lengths). It uses some of the scripts from https://github.com/tanghaibao/jcvi
I'll use it in the Maker tutorial I'm preparing for GTN

### CODE REVIEW

File: tools/jcvi_gff_stats/jcvi_gff_stats.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+        python -m jcvi.annotation.stats histogram 'input.gff'
+    ]]></command>
+    <inputs>
+        <param name="gff" type="data" format="gff" label="Annotation to analyse"/>
+        <param name="genome" type="data" format="fasta" label="Corresponding genome sequence"/>
+    </inputs>
+    <outputs>
+        <data format="txt" name="summary" label="${tool.name} on ${on_string}: summary"/>
+        <data format="pdf" name="gene_length" label="${tool.name} on ${on_string}: gene length" from_work_dir="Gene_Length.input.pdf"/>
```
Comment (bgruening) [inline tools/jcvi_gff_stats/jcvi_gff_stats.xml]: Would it make sense to stitch those reports together in on PDF?

**Comment by abretaud** [comment]
Done! I used the `all_fasta` data table instead of `fasta_indexes`, but not really sure which one is the best (I use all_fasta much more than fasta_indexes usually).

**Comment by abretaud** [comment]
@bgruening can I go ahead and merge or do you want to have another look before?

**Comment by bgruening** [comment]
Please go ahead!

**Pull request merged**